usaspending_api/common/retrieve_file_from_uri.py

```diff
@@ -60,6 +61,8 @@ def _handle_http(self):
         f = tempfile.SpooledTemporaryFile()
         f.write(r.content)
         f.seek(0)  # go to beginning of file for reading
+        if self.dest_file_path:
```

Comment:
Is it difficult to implement this for local and s3 files as well?

Also, what are your thoughts on, instead of a __init__ parameter, what if there was a `copy_file_object` (or maybe just `copy`) method that accepted the file name and copied that-a-way?  In my head, I want the __init__ to be just about the file object.

And if you really just wanted to deal with http for now, maybe raise a NotImplementedError exception for the other two so other developers don't spend time trying to figure out why their files aren't copying.